<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfTeamVisibilityPrefWidget</class>
 <widget class="QWidget" name="WkfTeamVisibilityPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>团队可见性</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="teamVisibilityGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>团队可见性 &amp;&amp; 颜色</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="teamTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>可见性</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>颜色</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="newTeamHorizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>新团队名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="teamNameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="addTeamPushButton">
          <property name="text">
           <string>新增团队</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>teamTableWidget</tabstop>
  <tabstop>teamNameLineEdit</tabstop>
  <tabstop>addTeamPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
